changelog updates

LisaDelaney opened the pull request.

Commit:
changelog updates

saadrahim commented on CHANGELOG.md:
```diff
@@ -1,280 +1,418 @@
 # Changelog for TransferBench
 
+Full documentation for TransferBench is available at
+[https://rocm.docs.amd.com/projects/TransferBench/en/latest/index.html](https://rocm.docs.amd.com/projects/TransferBench/en/latest/index.html).
```
```suggestion
[https://rocm.docs.amd.com/projects/TransferBench](https://rocm.docs.amd.com/projects/TransferBench).
```

LisaDelaney committed:
Update CHANGELOG.md

Co-authored-by: Saad Rahim (AMD) <44449863+saadrahim@users.noreply.github.com>

samjwu committed:
Standardize documentation (#59)

* Standardize documentation

Applies changes noted in https://github.com/RadeonOpenCompute/rocm-docs-core/issues/330

* Fix typo in conf.py

* Import re for conf.py

* Search EnvVars.hpp for version number instead of CMakeLists.txt

* Update Doxyfile output dir

* Change line endings to LF

Commit:
readme updates

LisaDelaney committed:
Merge branch 'develop' into readme-updates
